internal/config/config.go

```diff
@@ -88,17 +88,18 @@ type TagFilter struct {
 type RunningOutput struct {
 	Name   string
 	Output outputs.Output
+	Config *PrefixFilterConfig
 }
 
 type RunningPlugin struct {
 	Name   string
 	Plugin plugins.Plugin
-	Config *PluginConfig
+	Config *PrefixFilterConfig
 }
 
-// PluginConfig containing a name, interval, and drop/pass prefix lists
+// PrefixFilterConfig contains the name, interval (for plugin) and drop/pass prefix lists
 // Also lists the tags to filter
-type PluginConfig struct {
+type PrefixFilterConfig struct {
```

Comment:
Or if Name & Interval aren't being used from here (I'm not sure if they are?), remove them and just call this struct `Filter`